posthog/api/organization.py

```diff
@@ -171,6 +185,10 @@ def create(self, validated_data, **kwargs):
 
     def create_team(self, organization: Organization, user: User) -> Team:
         if self.enable_new_onboarding(user):
+            organization.setup_section_2_completed = (
```

Comment:
I'm not so sure about that, because we would now be calling `enable_new_onboarding()` twice.

Reply:
We already do call it twice. If calling it 3 times becomes a problem we can add caching.

Unclear code flow is going to be a much bigger issue here long-term.

Reply:
Fair enough! Can you give this a final look. We can't use the `kwargs` from `.bootstrap` because we need the user  object to be generated first (as we compute the feature flag with their `distinct_id`).